spotlessSelf.gradle

```diff
@@ -24,9 +24,12 @@ spotless {
 		trimTrailingWhitespace()
 	}
 	groovy {
-		target 'build.gradle', 'spotlessSelf.gradle', 'settings.gradle'
-		greclipseFormat().configFile('spotless.eclipseformat.xml')
-		paddedCell()
+		target fileTree('.') {
+			include '**/*.gradle'
+			exclude '_ext/**'
+		}
+		custom 'preventFormatPingPong', { return it.replaceAll('}[ \t\f]+}', '}}') }
```

Comment:
Please refactor the closure of the `custom 'noInternalDeps', { ... }` step in the `java { ... }` block above, into a variable, and pass that variable as a custom step to both the `java` and `groovy` blocks, like so:

```gradle
spotless {
	def noInternalDepsClosure = { source ->
		if (source.contains('import org.gradle.internal.') {
			throw new AssertionError("Accidental internal import")
		}
	}
	java {
		. . .
		custom 'noInternalDeps', noInternalDepsClosure
		. . .
	}
	groovy {
		. . .
		custom 'noInternalDeps', noInternalDepsClosure
		. . .
	}
}
```